releases/release-1.14/exception.yaml

```
@@ -9,4 +9,76 @@
   thread: 
   pull_requests:
   - 
-  status: 
+  status:
+  
+  - name: Support Out-of-Tree vSphere Cloud Provider 
+  issue: 670
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 30th, 2019
+  thread: https://groups.google.com/forum/#!topic/kubernetes-milestone-burndown/psJ0VI-gt5U
+  pull_requests:
+  - 
+  status: Approved 
+  
+  - name: Kubelet Resource Metrics Endpoint
+  issue: 727
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 31st, 2019
+  thread: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-milestone-burndown/pFGpJkGlOZU/YEjkMngtFAAJ
+  pull_requests: Will open after KEP merged 
+  - 
+  status: Approved 
+
+- name: Promote RunAsGroup  to beta
+  issue:213 
+  date_requested: Feb 1st, 2019
+  date_reviewed: Feb 4th, 2019
+  thread: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-milestone-burndown/d7P6XjunEjg/nHmWwaeWEwAJ
+  pull_requests:https://github.com/kubernetes/kubernetes/pull/73007,  https://github.com/kubernetes/enhancements/pull/800, https://github.com/kubernetes/website/pull/12297
+  - 
+  status:Approved
+ 
+ - name: Promote HugePages to GA
+  issue: 750
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 31st, 2019
+  thread: https://urldefense.proofpoint.com/v2/url?u=https-3A__groups.google.com_d_msgid_kubernetes-2Dmilestone-2Dburndown_CAHROWxTeDMaBfSbAzag9KKN5EAOFuEQRF-252BnFiXYo-2D9-253DraVQqkg-2540mail.gmail.com-3Futm-5Fmedium-3Demail-26utm-5Fsource-3Dfooter&d=DwMFaQ&c=lnl9vOaLMzsy2niBC8-h_K-7QJuNJEsFrzdndhuJ3Sw&r=jp67QQzRU2EkaBX4M5K9sdqcpSUKjuk9hPRw5RDty3g&m=81tj2WV26jyBeRiD_Fg7sowvMMjrJsJqJ-VnbJYd8qA&s=X-1bC-7BfT0_M_peFnWsjEFq3ONeMjcxf91dXbXFRRo&e=
+  pull_requests:https://github.com/kubernetes/kubernetes/issues/72651
+  - 
+  status: Approved
+  
+  - name: Pid Limiting
+  issue: 757
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 31st, 2019
+  thread: https://urldefense.proofpoint.com/v2/url?u=https-3A__groups.google.com_d_msgid_kubernetes-2Dmilestone-2Dburndown_CAHROWxQ27R-2DG25n-252BzDCuAw2ew6dLnqtez0Z3c-2DNbWvFJH-2DH24A-2540mail.gmail.com-3Futm-5Fmedium-3Demail-26utm-5Fsource-3Dfooter&d=DwMFaQ&c=lnl9vOaLMzsy2niBC8-h_K-7QJuNJEsFrzdndhuJ3Sw&r=jp67QQzRU2EkaBX4M5K9sdqcpSUKjuk9hPRw5RDty3g&m=SR_3nDzUiANMKnKG4i-Slmv7V8KF-CoZpZVu9LTsIdg&s=EU0TA1-h_riKT8G_wn3XE3-y4jYw6x0Ncv2O6BOX3gc&e=
+  pull_requests:https://github.com/kubernetes/kubernetes/pull/72076
```

```suggestion
  pull_requests:
  - https://github.com/kubernetes/kubernetes/pull/72076
```

should be an array, even when only a single pr is referenced